R/SimLongitudinalGSF.R

```diff
@@ -0,0 +1,188 @@
+
+#' Simulate Longitudinal Data from a  GSF Model
+#'
+#' @param times (`numeric`)\cr the times to generate observations at.
```

Comment:
I think would be worth to consider `roxytypes`, now that it is on CRAN :) (separate PR of course)